backend/src/main/java/com/team28/sidedish/controller/CategoryController.java

```diff
@@ -0,0 +1,44 @@
+package com.team28.sidedish.controller;
+
+import com.team28.sidedish.controller.dto.CategoryResponse;
+import com.team28.sidedish.service.CategoryService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@Tag(name = "categories", description = "카테고리 API")
+@RestController
+@RequiredArgsConstructor
+public class CategoryController {
+
+    private final CategoryService categoryService;
+
+    @Operation(summary = "카테고리 목록 조회",
+            description = "모든 카테고리 목록을 조회합니다.",
+            responses = {
+                @ApiResponse(responseCode = "200",
+                        description = "카테고리 목록 조회 성공",
+                        content = {
+                            @Content(
+                                    mediaType = "application/json",
+                                    array = @ArraySchema(schema = @Schema(implementation = CategoryResponse.class))
+                            )
+                        })
+            }
+    )
+    @GetMapping("/categories")
+    public ResponseEntity<List<CategoryResponse>> getCategories() {
+        List<CategoryResponse> categories = categoryService.findAll();
+
+        return ResponseEntity.ok().body(categories);
```

Comment:
지역변수 할당 없이 간단히 표현할 수 있을것 같네요.

```suggestion
        return ResponseEntity.ok().body(categoryService.findAll());
```

Reply:
인라인처리하는 편이 더 깔끔하겠네요! 감사합니다 👍